install/README.md

```diff
@@ -2,8 +2,9 @@
 
 This folder contains all necessary files for configuration of the host computer and installation of companion-docker.
 
-To use it, just run the following line in your terminal **as root**:
+To use it, just run the installation script in your terminal **as root**, like so:
 
 ```bash
+sudo su
 curl -fsSL https://raw.githubusercontent.com/bluerobotics/companion-docker/master/install/install.sh | bash
```

Comment:
it's better to have `sudo su && command`, it'll avoid copy and paste problems.

Reply:
nice!